AdornUI-documentation/card.html

```diff
@@ -0,0 +1,168 @@
+                                <p class="discount">20% off</p>
+                            </div>
+                            <div class="card-last-row flex-vt">
+                                <button class="btn btn-primary add-to-cart">ADD TO CART</button>
+                                <button class="btn btn-secondary remove-from-wishlist">REMOVE FROM WISHLIST</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61fcd840b57bf800152b721a" frameborder="0"></iframe>
+                <h3 class="component-subtitle">Cards with badges</h3>
+                <div class="component-container-hz">
+                    <div class="card__container flex-vt badge-card">
+                        <img class="card__image" src="images/prd-img.jpg" />
+                        <span class="badge-w-card">NEW</span>
+                        <h3 class="card__title">Olay Cream</h3>
+                        <p class="card__description">Lakme Absolute Perfect Radiance Brightening Day Cream 50 g, SPF 30,
+                            Daily Illuminating Face Moisturizer for Glowing Skin - With Glycerin & Niacinamide</p>
+                        <div class="product-price-wrapper flex-hz">
+                            <p class="discounted-price">₹ 639</p>
+                            <p class="discount">20% off</p>
+                        </div>
+                        <div class="ratings">
+                            <span>4.4</span>
+                            <span><i class="fa fa-star"></i></span>
+                        </div>
+                        <div class="card-last-row flex-hz">
+                            <div><i class="fa fa-heart add-to-wishlist"></i></div>
+                            <button class="btn btn-primary add-to-cart">ADD TO CART</button>
+                        </div>
+                    </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61fcd918b57bf800152b721d" frameborder="0"></iframe>
+                <h2 class="component-subtitle">Cards with Text Overlays</h2>
+                <div class="component-container-hz">
+                    <div class="card__container flex-vt card-overlay">
+                        <img class="card__image" src="images/prd-img.jpg" />
+                        <h3 class="card__title">Olay Cream</h3>
+                        <p class="card__description">Lakme Absolute Perfect Radiance Brightening Day Cream 50 g, SPF 30,
+                            Daily Illuminating Face Moisturizer for Glowing Skin - With Glycerin & Niacinamide</p>
+                        <div class="product-price-wrapper flex-hz">
+                            <p class="discounted-price">₹ 639</p>
+                            <p class="discount">20% off</p>
+                        </div>
+                        <div class="ratings">
```

Comment:
Rating can be moved within the previous flex container as there's enough space on that row.

<img width="284" alt="image" src="https://user-images.githubusercontent.com/9329352/152556280-1abe9101-8840-4778-a5ff-aed5c51d54b0.png">